### PR TITLE
Prow job update, license header checking added

### PR DESCRIPTION
**Pull request by radoslawc**

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -24,8 +24,16 @@ presubmits:
...
         args:
         - lint
+  - name: presubmit-nephio-license-header-check
+    decorate: true
+    run_if_changed: "^.*.go$"
+    spec:
+      containers:
+      - image: nephio/gotests:4
+        command:
+        - "/usr/local/bin/checklicense.sh"
```
Comment (gvbalaji) [inline .prow.yaml:39]: New line needed?

Comment (johnbelamaric) [reply .prow.yaml:39]: agreed - otherwise @radoslawc this LGTM

Comment (radoslawc) [reply .prow.yaml]: fixed

**Pull request merged**